modules/HomePage/components/Posts.tsx

```diff
@@ -1,4 +1,5 @@
 import withErrorHandler from '@modules/shared/components/HOC/WithErrorHandler/WithErrorHandler';
+import ImagePollView from '@modules/shared/components/organisms/ImagePollView/ImagePollView';
 import MiniSurveyView from '@modules/shared/components/organisms/MiniSurveyView/MiniSurveyView';
 import TextPollView from '@modules/shared/components/organisms/TextPollView/TextPollView';
```

Comment:
I think we agreed in the first meeting to  create a file named `index.tsx`  which will be the entry to all components 
so when we consume components `/Posts.tsx`  file

`import {ImagePollView, MiniSurveyView, TextPollView } from '@modules/shared/components/organisms'`.  

### Instead of 

```
import ImagePollView from '@modules/shared/components/organisms/ImagePollView/ImagePollView';
import MiniSurveyView from '@modules/shared/components/organisms/MiniSurveyView/MiniSurveyView';i
mport TextPollView from '@modules/shared/components/organisms/TextPollView/TextPollView';
```

For readability reasons and also autocomplete for components inside the `index.tsx` file